django_fsm/__init__.py

```diff
@@ -264,7 +264,10 @@ def deconstruct(self):
         return name, path, args, kwargs
 
     def get_state(self, instance):
-        return instance.__dict__[self.name]
+        if self.name in instance.__dict__:
+            return instance.__dict__[self.name]
+
+        return getattr(instance, self.name)
```

Comment:
How do you get the situation when `self.name not in instance.__dict__` but getattr call succeed?